core-api/src/main/java/com/optimizely/ab/config/audience/match/NullMatch.java

```diff
@@ -16,20 +16,14 @@
  */
 package com.optimizely.ab.config.audience.match;
 
-import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+public class NullMatch extends Throwable {
```

Comment:
create two new exceptions with the suffix Exception and set the appropriate errorDescription